### PR TITLE
Fix: Missing field name translation in grid cell editor

### PR DESCRIPTION
**Pull request by blankse**
This PR translates the field name like in edit helper:
https://github.com/pimcore/pimcore/blob/8b21bdd0e5cbb42b4261e2e596849f57292d8529/bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/edit.js#L134-L137

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/gridCellEditor.js [hunk 1]
```diff
@@ -51,13 +51,16 @@ Ext.define('pimcore.object.helpers.gridCellEditor', {
...
-        var title = fieldInfo.label ? fieldInfo.label : fieldInfo.key;
-
         this.context = this.editingPlugin.context;
         // this.callParent(arguments);
 
         var tagType = fieldInfo.layout.fieldtype;
 
+        // translate title
+        if(typeof fieldInfo.layout.title != "undefined") {
+            fieldInfo.layout.title = t(fieldInfo.layout.title);
```
Comment (weisswurstkanone) [inline bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/gridCellEditor.js]: shouldn't it be ts instead ?

Comment (blankse) [reply bundles/AdminBundle/Resources/public/js/pimcore/object/helpers/gridCellEditor.js]: `ts()` is only a alias for `t()`
https://github.com/pimcore/pimcore/blob/8b21bdd0e5cbb42b4261e2e596849f57292d8529/bundles/AdminBundle/Resources/public/js/pimcore/functions.js#L64-L66

So i thought `t()` should be preferred

**Comment by blankse** [comment]
@brusch This change is not enough. So only the title of the Window is translated. But not the field itself.

**Comment by brusch** [comment]
You are absolutely right, thanks!

**Pull request merged**